PrepInsta Top 100 Codes/Armstrong_Number.java

```diff
@@ -0,0 +1,37 @@
+import java.util.Scanner;
+
+public class Armstrong_Number {
+     public static void main(String[] args) {
+        // first we have to give a input number using the Scanner class also we have to import the package of Scanner class
```

Comment:
Try to make your comment short and efficient

Reply:
ok sure